app/src/main/java/lt/vilnius/tvarkau/fragments/NewReportFragment.kt

```diff
@@ -440,38 +439,51 @@ class NewReportFragment : BaseFragment(),
                     val geocoder = Geocoder(context)
                     val place = PlacePicker.getPlace(context, data)
 
-                    var addresses = listOf<Address>()
+                    val addresses = mutableListOf<Address>()
 
                     place.latLng?.let {
                         locationCords = it
 
                         try {
-                            addresses = geocoder.getFromLocation(it.latitude, it.longitude, 1)
-                        } catch (e: IOException) {
-                            Timber.e(e)
+                            geocoder.getFromLocation(it.latitude, it.longitude, 1)?.let {
+                                addresses.addAll(it)
+                            }
+                        } catch (e: Throwable) {
+                            Timber.e(GeocoderException(e))
```

Comment:
I guess there will be various IO exceptions as stated in documentation.
How lat/lon would help us in debugging?